debian/changelog

```diff
@@ -1,3 +1,75 @@
+citus-enterprise (9.1.1.citus-1) stable; urgency=low
+
+  * Adds extensions to distributed object propagation infrastructure
+
+  * Adds node draining
+
+  * Adds support for ALTER ROLE propagation
+
+  * Adds support for aggregates in create_distributed_function
+
+  * Adds support for expressions in reference joins
+
+  * Adds support for returning RECORD in multi-shard queries
+
+  * Adds support for simple IN subqueries on unique cols in repartition joins
+
+  * Adds support for subqueries in HAVING clauses
+
+  * Automatically distributes unary aggs w/ combinefunc and non-internal stype
+
+  * Disallows distributed func creation when replication_model is 'statement'
+
+  * Drops support for deprecated real-time and router executors
+
+  * Fixes a bug causing SQL-executing UDFs to crash when passing in DDL
+
+  * Fixes a bug in local execution that could cause missing rows in RETURNING
+
+  * Fixes a bug that caused column_to_column_name to crash for invalid input
+
+  * Fixes a bug that caused inserts into local tables w/ dist. subqueries to crash
+
+  * Fixes a bug that caused maintenance daemon to fail on standby nodes
+
+  * Fixes a bug that caused other CREATE EXTENSION commands to take longer
+
+  * Fixes a bug that caused some noop DML statements to fail
+
+  * Fixes a bug that prevented REFRESH MATERIALIZED VIEW
+
+  * Fixes a bug that prevents dropping reference table columns
+
+  * Fixes a bug when view is used in modify statements
+
+  * Fixes a crash in IN (.., NULL) queries
+
+  * Fixes a crash when calling a distributed function from PL/pgSQL
+
+  * Fixes a memory leak in adaptive executor when query returns many columns
+
+  * Fixes an issue that caused CTEs to sometimes leak connections
+
+  * Fixes an issue that caused failures in RHEL 6 builds
+
+  * Fixes potential segfault in standard_planner inlining functions
+
+  * Fixes queries with repartition joins and group by unique column
+
+  * Fixes strange errors in DML with unreachable sublinks
+
+  * Fixes underflow init of default values in worker extended op node creation
+
+  * Prevents statements in SQL functions to run outside of a transaction
+
+  * Improves CTE/Subquery performance by pruning intermediate rslt broadcasting
+
+  * Removes citus.worker_list_file GUC
+
+  * Revokes usage from the citus schema from public
+
+ -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Fri, 20 Dec 2019 15:12:9 +0000
```

Comment:
```
parsechangelog/debian: warning:     debian/changelog(l71): badly formatted trailer line
LINE:  -- Onur Tirtir <Onur.Tirtir@microsoft.com>  Fri, 20 Dec 2019 15:12:9 +0000
```

I suggest you update the timestamp manually.

(Note: adding this problem to the issue https://github.com/citusdata/tools/issues/57)